apps/website/versioned_docs/version-v3.x/supported-networks/supported-networks.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Supported Networks
+description: Supported Networks
+sidebar_label: Supported Networks
+sidebar_position: 1
+---
+
+MACI has been tested on the following EVM networks:
```

Comment:
we can add a note this was done with v2 but we expect similar results (to follow up)